app/org/sagebionetworks/bridge/play/controllers/AuthenticationController.java

```diff
@@ -55,9 +56,13 @@ public Result verifyEmail() throws Exception {
         JsonNode json = requestToJSON(request());
         EmailVerification emailVerification = parseJson(request(), EmailVerification.class);
         Study study = getStudyOrThrowException(json);
+        ScheduleContext context = new ScheduleContext.Builder()
+                .withStudyIdentifier(study)
+                .withClientInfo(getClientInfoFromUserAgentHeader()).build();
```

Comment:
What do we need from the ScheduleContext other than ClientInfo? As is, the ScheduleContext is incomplete, notably lacking dataGroups. I'm concerned someone in the future might forget that the ScheduleContext is incomplete and try to code against it and run into trouble because it's incomplete. For type safety, it would be better to just pass ClientInfo directly.